bundles/org.openhab.binding.growatt/src/main/java/org/openhab/binding/growatt/internal/cloud/GrowattCloud.java

```diff
@@ -189,6 +189,10 @@ private Map<String, JsonElement> doHttpRequestInner(HttpMethod method, String en
             request.content(new FormContentProvider(fields), FORM_CONTENT);
         }
 
+        if (logger.isTraceEnabled()) {
```

Comment:
I'm curious, what is the benefit of enclosing logger.trace line in a logger.IsTraceEnabled block?

Reply:
Ok

Reply:
@lsiepel just for info: I am working with a guy on the forum to add support for his inverter type, so I am in process of making some changes.